src/hazelcore/Statics_Typ.re

```diff
@@ -0,0 +1,19 @@
+let rec syn = (ctx: Contexts.t, ty: HTyp.t): Kind.t =>
+  switch (ty) {
+  | TyVar(idx, _) =>
+    let (_, k) = List.nth(ctx.tyvars, idx);
```

Comment:
Define this as a function TyVarCtx, rather than assuming it will always be a list